sdk/iot/core/inc/az_iot_core.h

```diff
@@ -18,6 +18,17 @@
 
 #include <_az_cfg_prefix.h>
 
+/**
+ * @brief Azure IoT service MQTT connection properties.
+ *
+ */
+enum
+{
+  AZ_CLIENT_DEFAULT_MQTT_CONNECT_PORT = 8883,
```

Comment:
Naming: I don't think this should be the "Default" port. Instead it could be the "port" since it cannot change.